backend/models/Link.js

```diff
@@ -0,0 +1,38 @@
+module.exports = function(sequelize, Sequelize) {
+  let Link = sequelize.define(
+    "links",
+    {
+      id: {
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        primaryKey: true
+      },
+      siteName: {
+        type: Sequelize.STRING
+      },
+
+      linkTitle: {
+        type: Sequelize.STRING
+      },
+      url: {
+        type: Sequelize.STRING
+      },
+      storyTags: {
+        type: Sequelize.STRING
+      },
+      timesClicked: {
+        type: Sequelize.INTEGER
+      }
+    },
```

Comment:
Also need to add a `Topic` field (I think this is either a separate data model or an enumerated string, not sure)

Reply:
changed Link fields to match spreadsheet